### ISSUE TITLE
Player controls not displaying when hovering over taskbar

### ISSUE THREAD

**Issue by roryhewitt** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#193]
On occasion, when I hover over the GPMDP icon on the taskbar, the player controls (Previous/Play/Next) don't appear - all I see is the name of the track/artist (same as I would see if I was using a browser window to use Google Play). When this happens, if I click on another taskbar icon (to maximize that application's window) and then click it again (to minimize that window), then the hover controls appear again next time I hover over the GPMDP icon.

![image](https://cloud.githubusercontent.com/assets/11558891/13446895/fd9424a2-dfcb-11e5-9474-6c6d8f83e17c.png)

It may be due to the fact that I have minimized the GPMDP window, but I'm not sure if that's consistently the problem.

I'm using v2.1.1 on Windows 7 Professional (x64).

**Comment by MarshallOfSound** [comment]
That is windows functionality and nothing to do with me :)

Windows determines whether to show the tooltip or the aero preview (with the button controls) depending on how you interact with the taskbar.

TLDR: Nothing I can do

**Comment by roryhewitt** [comment]
Hey Samuel,

Thanks for the speedy response!

Do you happen to know what their 'rules' are about this? I don't think I'm
interacting with the toolbar in any particular way, but sometimes it works
and sometimes it doesn't.

If you happen to have a link handy to a description of this functionality,
I'd love to see it. Of course, if you're busy, Google is my friend.

Thanks again,

Rory

p.s. The app is great!

**Comment by MarshallOfSound** [comment]
I'm sure there are any documented rules persay, but it was definitely a thing in Windows 7 (not sure what OS you are on) where it would randomly choose between tooltip and preview.  Aero was still a relatively new thing back then :+1:

**Comment by roryhewitt** [comment]
Hey Samuel,

Hmmm. OK, it doesn't appear to be an Aero-specific thing; if I turn off Aero completely, and just go back to "Windows Basic" (I'm  on Windows 7 Enterprise), I can still get some basic controls:

![image](https://cloud.githubusercontent.com/assets/11558891/13468149/3b13262c-e056-11e5-9dcc-23ce1a93fca3.png)

But these only appear sporadically. I can consistently get them to appear by clicking another taskbar icon twice (to maximize/minimize that window). But as soon as I open another window, I just get the tooltip, with no controls.

Of course, it may still be some Microsoft weirdness that you can't control - I'm just saying it's not specific to Aero.
